addons/mail/static/src/new/core/thread_service.js

```diff
@@ -437,7 +442,9 @@ export class ThreadService {
             ]);
 
             if (serverData.last_interest_dt) {
-                thread.lastInterestDateTime = luxon.DateTime.fromISO(serverData.last_interest_dt);
+                thread.lastInterestDateTime = luxon.DateTime.fromISO(
+                    new Date(serverData.last_interest_dt).toISOString()
+                );
```

Comment:
why this double convert?

Reply:
`fromISO(serverDate.last_interest_dt)` was not properly parsed, because not an ISO.
I fixed it, as it was required by test to make it pass.